SECURITY.md

```diff
@@ -0,0 +1,64 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+We take the security of Boston Open Source's website and community infrastructure seriously. If you discover a security vulnerability, please follow these steps:
+
+1. **DO NOT** create a public GitHub issue
+2. Report via one of these secure channels (in order of preference):
+   - Send a private message on [Zulip](https://osdc.zulipchat.com/#narrow/stream/406743-boston) to the maintainers
+   - Use GitHub's private security advisory reporting
+   - Direct message on [Mastodon](https://floss.social/@bostonopen)
+3. Provide as much information as possible about:
+   - The location and nature of the vulnerability
+   - Steps to reproduce (if applicable)
+   - Potential impact of the vulnerability
+
+## What to Report
+
+Please report issues related to:
+
+- Cross-site scripting (XSS) vulnerabilities
+- Exposed sensitive information
+- Issues with authentication (if implemented)
+- Problems with data handling
+- Concerns about personal information exposure
+- Any other security-related issues
+
+## Response Timeline
+
+- Initial Response: Within 48 hours
+- Status Update: Within 5 days
+- Resolution Timeline: Case-by-case basis depending on severity
+
```

Comment:
```suggestion
```

I'm suggesting we delete this whole "response timeline" section because I can't personally guarantee that I will be this responsive. I don't want to set expectations too high.